### PR TITLE
composer: allow PHP 8.0

### PR DESCRIPTION
**Pull request by TomasVotruba**

### CODE REVIEW

**Review by Jean85 — CHANGES_REQUESTED** [review]
Thank you for your contribution! Could you please add PHP 8 in the CI matrix to test it out?

**Comment by TomasVotruba** [comment]
Sure

**Comment by TomasVotruba** [comment]
What's the current PHP on travis?

I'm used to Github Actions last year and there the version works fine.
Would consider switching to Github Actoins? I can handle it

**Comment by Jean85** [comment]
Happy to accept a (separate) PR that migrates to GitHub Actions, I never used them so I don't know where to start.

File: .github/workflows/tests.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    push:
+        branches:
+            - master
+
+jobs:
+    tests:
+        runs-on: ubuntu-latest
+        strategy:
+            matrix:
+                php: ['7.0', '7.1', '7.2', '7.3', '7.4', '8.0']
```
Comment (TomasVotruba) [inline .github/workflows/tests.yaml:14]: Btw, what about bumping min PHP version? PHP 7.1 and lower are dangerous to use - https://www.php.net/supported-versions.php

Also, package will be still available for the older PHP versions with version 1.5, so there is no BC break

**Comment by TomasVotruba** [comment]
Ah, just saw *separate* word (brackets)

I'll make a new PR

**Comment by TomasVotruba** [comment]
There it is: https://github.com/Jean85/pretty-package-versions/pull/26

After that gets merged, I'll rebase this PR

**Comment by Jean85** [comment]
#26 merged and migration is completed thanks to #27. Thank you for your help! Now you can rebase...

**Comment by TomasVotruba** [comment]
Rebased... it seems PHPUnit is the problem:

image

**Comment by Jean85** [comment]
It's not PHPUnit but Prophecy specifically: https://github.com/phpspec/prophecy/issues/491

**Comment by TomasVotruba** [comment]
Yea, fractal of bad version design :/

Could you merge this, as it's only dev dependency and unrelated to version?

I had to fork this package, as I needed to continue upgrading to PHP 8.

I'd prefer to use original code though.

**Comment by Jean85** [comment]
Closing in favor of #28

**Comment by TomasVotruba** [comment]
Nice detailed CI checks :+1: 

Thank you

**Pull request closed without merging**